ext/rav1e.cmd

```diff
@@ -11,7 +11,7 @@
 : # Also, the error that "The target windows-msvc is not supported yet" can safely be ignored provided that rav1e/target/release
 : # contains rav1e.h and rav1e.lib.
 
-git clone -b v0.3.1 --depth 1 https://github.com/xiph/rav1e.git
+git clone -b 0.3 --depth 1 https://github.com/xiph/rav1e.git
```

Comment:
Why is this change in this commit? It seems unrelated to the CI update.

Reply:
I think this is to fix an issue with rav1e compiling

https://github.com/xiph/rav1e/pull/2324

https://github.com/xiph/rav1e/pull/2325

Reply:
Ah, interesting. I imagine these fixes will land in rav1e v0.4 soon. I will probably wait until rav1e v0.4 hits before accepting this PR, but I will accept it shortly after that.

Reply:
Actually, I've changed my mind. I'll pull this now in order to get the CIs back up and running. Thanks for the information.